.github/sync/RELEASE.md

```diff
@@ -364,9 +378,15 @@ Currently, there are only 2 activities we use rever for, (1) aggregating the aut
 
 </details>
 
-## 8. Open PRs to bump [Anaconda Recipes][Anaconda Recipes] and [conda-forge][conda-forge] feedstocks to use `YY.M.0`.
+## 9. Open PRs to bump [Anaconda Recipes][Anaconda Recipes] and [conda-forge][conda-forge] feedstocks to use `YY.M.0`.
+
+> **Note**
+> Conda-forge's PRs will be auto-created via the `regro-cf-autotick-bot`. If any changes need to be made to the recipe that were not automatically added, create a new branch based off of autotick's branch (autotick's branches usually use the `regro-cf-autotick-bot:XX.YY.0_[short hash]` syntax), add any changes via commits to that new branch, and then open a new PR and push it against the `main` branch (`regro-cf-autotick-bot` will close the auto-created PR once the new PR is merged).
```

Comment:
Since maintainers can push changes directly to the autotick branch the above is only necessary if the release pilot/copilot are not maintainers of the conda-forge feedstock, dunno if we want to highlight this here? Could also link to conda-forge docs: https://conda-forge.org/docs/maintainer/updating_pkgs.html#pushing-to-regro-cf-autotick-bot-branch